utils/rdf_convertor.py

```diff
@@ -14,6 +14,9 @@ def __init__(
         self.remove_old_files = remove_old_files
 
     def convert_files(self):
+        print("inside convert files")
```

Comment:
I guess this are debugging lines. Would be good if you just comment them or remove them entirely.

Reply:
> I guess this are debugging lines. Would be good if you just comment them or remove them entirely.

yea, I've cleaned the code now.